src/lib.rs

```diff
@@ -29,30 +29,36 @@ pub fn App() -> impl IntoView {
   provide_i18n_context();
   provide_query_client();
 
+  let ui_theme = create_rw_signal::<String>(String::from("retro"));
+  provide_context(ui_theme);
+
+  let authenticated = create_rw_signal::<bool>(false);
```

Comment:
this or something like this will be needed to send a signal to the rest of the page that we are logged in when the pages is Hydrated. so we do need this

Reply:
The site is a blocking resource. It will have already been fetched by the time the HTML is returned to the browser.

Reply:
if the site query can be used as a signal then that should work. 

will remove this as it can't work in SSR only mode anyway.